orca-queue/src/test/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandlerTest.kt

```diff
@@ -1839,6 +1831,70 @@ object RunTaskHandlerTest : SubjectSpek<RunTaskHandler>({
       }
     }
   }
+
+  describe("when a stage had skipped manual judgment stage with auth propagated") {
+    given("a stage with a manual judgment type and auth propagated") {
+      val timeout = Duration.ofMinutes(5)
+      val lastModifiedUser = StageExecution.LastModifiedDetails()
+      lastModifiedUser.user = "user2"
+      val pipeline = pipeline {
+        stage {
+          refId = "1"
+          type = "manualJudgment"
+          status = SUCCEEDED
+          requisiteStageRefIds = setOf("1")
```

Comment:
This is a circular dependency, right?  I don't think it's necessary...

Reply:
Removed